1/a-burke/project1.py

```diff
@@ -0,0 +1,105 @@
+#Author: Ally Burke
+
+import sys
+sys.path.append('../../extra-files')
+sys.path.append('../../0/a-burke')
+from molecule import Molecule
+import numpy as np
+from numpy import linalg as la
+from masses import mass
+from masses import charge
+import numpy.lib.scimath as nls
+import scipy.constants as sci
```

Comment:
Scipy.constants seems like a very useful package! I'm a bit disappointed I didn't know about it when I was doing the programming projects.